tests/crds_test.go

```diff
@@ -117,3 +118,42 @@ func TestCRDsWithUnderscore(t *testing.T) {
 
 	execCrd2Pulumi(t, "python", "crds/underscored-types/networkpolicy.yaml", validateUnderscore)
 }
+
+func TestKubernetesVersionNodeJs(t *testing.T) {
+	validateVersion := func(t *testing.T, path string) {
+		// enter and build the generated package
+		withDir(t, path, func() {
+
+			require.NoError(t, exec.Command("npm", "i").Run())
```

Comment:
nit: can we use the long form `install` here to make it self-documenting for other developers who may not be familiar with NodeJS/npm?

Reply:
Should we also get the output of the npm command to log in case there is an error? Would help with debugging.

Reply:
Sure; I made a helper method to only dump the output on error.